cmd/cluster-init/main.go

```diff
@@ -235,10 +242,19 @@ func initClusterBuildFarmDir(o options) error {
 	return nil
 }
 
-func buildFarmDirFor(releaseRepo string, clusterName string) string {
+func buildFarmDirFor(releaseRepo, clusterName string) string {
 	return filepath.Join(releaseRepo, "clusters", "build-clusters", clusterName)
 }
 
 func serviceAccountKubeconfigPath(serviceAccount, clusterName string) string {
 	return fmt.Sprintf("sa.%s.%s.config", serviceAccount, clusterName)
 }
+
+func appendIfNotContains(slice []string, item string) []string {
```

Comment:
I had considered converting it to a set, but wasn't sure what the best practice there was. If it's standard to just use `sets.NewString` I will do so.

Reply:
Actually, doing so results in a new ordering of all the configs we touch. It is deterministic, so we could just run in `update` mode and commit the new ordering. Thoughts?

Reply:
I see - the default behavior of `.List()` is to sort. Likely useful to have these sorted so that new additions are deterministic (they would be order-dependent today).